src/Components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useContext,
+} from 'react';
+import cn from 'classnames';
+import { TodoContext } from '../../TodoContext';
+import { Todo } from '../../types/Todo';
+
+export const TodoItem: React.FC<{ todo: Todo }> = React.memo(({ todo }) => {
+  const {
+    isLoading,
+    editingTodo,
+    handleEditing,
+    setHandleEditing,
+    setIsChosenToRename,
+    setEditingTodo,
+    handleDelete,
+    makeTodoCompleted,
+  } = useContext(TodoContext);
+
+  return (
+    <div
+      data-cy="Todo"
+      className={cn('todo', {
+        completed: todo.completed,
+      })}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          onChange={() => null}
+          onClick={() => {
+            makeTodoCompleted(todo.id, todo.completed);
+          }}
```

Comment:
use `onChange` intread of `onClick`

move ` makeTodoCompleted()` to ` onChange`